crates/telio-nurse/src/rtt/ping.rs

```diff
@@ -38,8 +40,40 @@ impl Ping {
     /// * `no_of_tries` - How many pings should be sent.
     pub fn new(no_of_tries: u32) -> std::io::Result<Self> {
         Ok(Self {
-            client_v4: Arc::new(Client::new(&Config::default())?),
-            client_v6: Arc::new(Client::new(&Config::builder().kind(ICMP::V6).build())?),
+            client_v4: Arc::new(Client::new(
```

Comment:
nit: Can you combine these builders into a function which takes in IPv4/IPv6 as parameter ?

Reply:
Done